app/src/main/java/com/infomaniak/mail/views/AvatarView.kt

```diff
@@ -167,4 +191,12 @@ class AvatarView @JvmOverloads constructor(
             )
         }
     }
+
+    fun loadAvatar(correspondent: Correspondent?, bimi: Bimi?) {
+        if (bimi == null || !bimi.isCertified) {
+            loadAvatar(correspondent)
+        } else {
+            loadBimiAvatar(ApiRoutes.bimi(bimi.svgContentUrl.orEmpty()), correspondent)
```

Comment:
If `bimi.svgContentUrl` is empty, we should call `loadAvatar(correspondent)`right ?